core-bundle/src/Command/MigrateCommand.php

```diff
@@ -0,0 +1,278 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Command;
+
+use Contao\CoreBundle\Framework\ContaoFramework;
+use Contao\CoreBundle\Migration\Migrations;
+use Contao\InstallationBundle\Database\Installer;
+use Symfony\Component\Config\Exception\FileLocatorFileNotFoundException;
+use Symfony\Component\Config\FileLocator;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Filesystem\Filesystem;
+
+class MigrateCommand extends Command
+{
+    /**
+     * @var Migrations
+     */
+    private $migrations;
+
+    /**
+     * @var FileLocator
+     */
+    private $fileLocator;
+
+    /**
+     * @var string
+     */
+    private $projectDir;
+
+    /**
+     * @var ContaoFramework
+     */
+    private $framework;
+
+    /**
+     * @var ?Installer
+     */
+    private $installer;
```

Comment:
did you see there is an `Installer` in the core-bundle as well? Not exactly sure why, but I think we moved it from the installation bundle to be always available. Or maybe to opposite 🙈 🙃

Reply:
I don’t think they do exactly the same.